docs/encrypt-binary-relay-log-files.md

```diff
@@ -0,0 +1,40 @@
+# Encrypt binary log files and relay log files
```

Comment:
This is an upstream variable. Why do we have a separate section for it in Percona Docs?

Reply:
One of our documentation initiatives is that we have complete docs.